script.py

```diff
@@ -0,0 +1,78 @@
+# %% [markdown]
+# ###Preprocessing
+
+# %%
+import pandas as pd
+import numpy as np
+import warnings
+
+# %%
+my_data = pd.read_csv("TestData.csv")
+
+# %%
+my_data.head()
+
+# %%
+my_data.isna().sum()
+
+# %%
+my_data.rename(columns={'Resale Value': 'Resale Value ($)', 'Resale Per Ticket': 'Resale Per Ticket ($)', 'Resale Difference': 'Resale Difference ($)'}, inplace=True)
+
+# %%
+my_data.dtypes
+
+# %%
+my_data.head()
+
+# %%
+warnings.filterwarnings("ignore", message="The default value of regex will change from True to False in a future version.", category=FutureWarning)
+my_data["Resale Value ($)"] = my_data["Resale Value ($)"].str.replace('$', '')
+my_data["Resale Per Ticket ($)"] = my_data["Resale Per Ticket ($)"].str.replace('$', '')
+my_data["Resale Difference ($)"] = my_data["Resale Difference ($)"].str.replace('$', '')
+
+# %%
+my_data['Resale Value ($)'] = my_data['Resale Value ($)'].str.replace(',', '').astype(float)
+my_data["Resale Per Ticket ($)"] = my_data["Resale Per Ticket ($)"].str.replace(',', '').astype(float)
+my_data["Resale Difference ($)"] = my_data["Resale Difference ($)"].str.replace(',', '').astype(float)
```

Comment:
When I attempt to run your code, I get a ValueError on line 36. If you are running this code successfully without this error, could you reply with a screenshot of your output? Otherwise, please resolve and push changes.